com.reprezen.swagedit.feature/feature.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <feature
       id="com.reprezen.swagedit.feature"
-      label="SwagEdit"
+      label=" KaiZen-OpenAPI-Editor"
```

Comment:
Does this label need to be machine-readable, without whitespace?  If it's a label for humans, please replace hyphens with spaces.